addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/discovery/FreeboxDiscoveryService.java

```diff
@@ -142,6 +146,27 @@ public void onDataFetched(ThingUID bridge, LanHostsConfig hostsConfig) {
                 }
             }
         }
+
+        if (airPlayDevices != null) {
+            // AirPlay devices
+            for (AirMediaReceiver device : airPlayDevices) {
+                String name = device.getName();
+                Boolean videoCapable = device.isVideoCapable();
+                logger.debug("AirPlay Device name {} video capable {}", name, videoCapable);
+                if ((name != null) && !name.isEmpty() && Boolean.TRUE.equals(videoCapable)) {
+                    String uid = name.replaceAll("[^A-Za-z0-9_]", "_");
+                    thingUID = new ThingUID(FreeboxBindingConstants.FREEBOX_THING_TYPE_AIRPLAY, bridge, uid);
+                    if (thingUID != null) {
```

Comment:
You're right, this test is stupid !
After a search, I found similar test in the following bindings: NTP, DSC alarm, Harmonyhub, max, minecrast, network, opensprinkler, RFXCOM, smaenergymeter, tellstick and vibrotonic.